string jig
string spot

---------------
bool success
jig_tape_data data
